tensorflow_addons/callbacks/time_stopping.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Callback that stops training when a specified amount of time has passed."""
+
+from __future__ import absolute_import, division, print_function
+
+import datetime
+import time
+
+import tensorflow as tf
+from tensorflow.keras.callbacks import Callback
+
+
+@tf.keras.utils.register_keras_serializable(package='Addons')
+class TimeStopping(Callback):
+    """Stop training when a specified amount of time has passed.
+
+    Args:
+        seconds: maximum amount of time before stopping.
+            Defaults to 86400 (1 day).
+        verbose: verbosity mode. Defaults to 0.
```

Comment:
Is there a reason for using an expect int here instead of a boolean? Doesn't seem like varying verbosity values will make any difference, so better to prevent that confusion

Reply:
I use int here for consistency with EarlyStopping Callback.
https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/keras/callbacks.py#L1134-L1251
And I would assume they also use int instead of just boolean is so that it is consistent with other places that has verbose as a parameter. This is my reasoning (for consistency), but if you feel like changing to boolean is better let me know! Thanks for the feedback!

Reply:
Thanks. Still feels a little intuitive, but I agree better to stay with the convention.